includes/admin/home/tasks/task/class-sensei-home-task-pro-upsell.php

```diff
@@ -0,0 +1,123 @@
+<?php
+/**
+ * File containing the Sensei_Home_Task_Pro_Upsell class.
+ *
+ * @package sensei-lms
+ * @since $$next-version$$
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Sensei_Home_Task_Pro_Upsell.
+ *
+ * @since $$next-version$$
+ */
+class Sensei_Home_Task_Pro_Upsell implements Sensei_Home_Task {
+
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+		add_filter( 'sensei_home_tasks', [ $this, 'maybe_add_external_icon_property' ] );
+	}
+
+	/**
+	 * Maybe add external icon to the task.
+	 *
+	 * @param array $tasks Tasks.
+	 *
+	 * @return array
+	 */
+	public function maybe_add_external_icon_property( $tasks ) {
+		return array_map(
+			function ( $task ): array {
+				if ( self::get_id() === $task['id'] ) {
+						$task['external_icon'] = true;
```

Comment:
The same I commented on the React component would apply to the property here, if you agree.

Reply:
Thanks, updated here https://github.com/Automattic/sensei/pull/7562/commits/e7de6d6015a3966431742255ef0dbb7169311d1d